src/nprintml/label/aggregator/pcap.py

```diff
@@ -80,24 +76,24 @@ def merge_npt(self):
         npt_paths = []
         largest_npt = None
         file_count = 0
-        npt_files = (npt_file for npt_file in self.npt_csv.iterdir() if npt_file.is_file())
-        
+        npt_files = (npt_file for npt_file in self.npt_csv.rglob('*') if npt_file.is_file())
+
         for (file_count, npt_file) in enumerate(npt_files, 1):
             npt = self.load_npt(npt_file)
 
             if largest_npt is None or npt.shape[0] > largest_npt.shape[0]:
                 largest_npt = npt
 
             npts0.append(npt)
-            npt_paths.append(str(npt_file))
+            npt_paths.append(str(npt_file.relative_to(self.npt_csv).with_suffix('')))
```

Comment:
This fixes (the insanity of) the user having to specify the _full_ path to the PCAP file in the labels index, (especially since here it's actually the path to the nPrint file, and as such unknown to the user); and, this discards the suffix (.npt) to match the above.